auction9-backend/serverless.yml

```diff
@@ -57,7 +57,16 @@ functions:
           method: post
           cors: true
 
-  # return all my auctions
+  # returns auction bids (users' bids history)
+  getAuctionBids:
+    handler: handler.getAuctionBids
+    events:
+      - http:
+          path: auctionBids/{auctionID}
```

Comment:
Let's change the path to `auctions/{id}/bids

Reply:
Sure, I'll change it.